src/utils/components/NetworkInterfaceModal/utils/helpers.ts

```diff
@@ -35,6 +37,28 @@ export const updateVMNetworkInterfaces = (
   return updatedVM;
 };
 
+export const patchVMNetworkInterfaces = (
```

Comment:
Please change the function name to updateVM...

Reply:
Sure done